pom.xml

```diff
@@ -2,13 +2,19 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.neo4j</groupId>
     <artifactId>neo4j-jdbc</artifactId>
-    <version>2.3-SNAPSHOT</version>
+    <version>2.0.5-SNAPSHOT</version>
```

Comment:
OK, if I'd read the PR comment before jumping to the code, I would have had my answer... I guess no version greater than 2.0.x was ever published.